LandSandBoat/server
# [core] Audit blacklist commands

**_I affirm:_**

## What does this pull request do?

* Rename blacklist packets
* Add support for client requesting blacklist
* Fix issue with blacklists only working on last 12 (page size 12) blacklisted characters

## Steps to test these changes

blist < 12 characters, see it work
blist 12 characters, see it work
blist > 12 characters, see it work
